test-suite/Math/NumberTheory/ArithmeticFunctionsTests.hs

```diff
@@ -55,13 +55,12 @@ divisorsProperty4 (NonZero n) = S.toAscList (runFunction divisorsA n) == IS.toAs
 divisorsProperty5 :: NonZero Int -> Bool
 divisorsProperty5 (NonZero n) = S.toAscList (runFunction divisorsA n) == sort (runFunction divisorsListA n)
 
--- | 'divisorsListInRange' matches 'divisorsA' with a filter
-divisorsProperty6 :: Positive Int -> NonNegative Int -> NonNegative Int -> Bool
-divisorsProperty6 (Positive a) (NonNegative b) (NonNegative c) = runFunction (divisorsInRangeA from to) n == expected
-  where from = a
-        to = from + b
-        n = to + c
-        expected = filter (\d -> d >= from && d <= to) $ S.toAscList (runFunction divisorsA n)
+-- | 'divisorsInRange' matches 'divisorsA' with a filter
+divisorsProperty6 :: Positive Int -> NonNegative Int -> Bool
+divisorsProperty6 (Positive a) (NonNegative b) = runFunction (divisorsInRangeA to) n == expected
+  where to = a
+        n = to + b
+        expected = S.takeWhileAntitone (<=to) (runFunction divisorsA n)
```

Comment:
`S.takeWhileAntitone` is a comparably new addition to `containers`, which causes [GHC 8.0.2 build](https://travis-ci.org/github/Bodigrim/arithmoi/jobs/666447515) to fail.
Please use `S.split` or `S.splitMember` instead.